netjson-monitoring.lua

```diff
@@ -236,45 +252,123 @@ function is_excluded(name)
   end
 end
 
+function find_default_gateway(routes)
+  for i = 1, #routes do
+    if routes[i].target == '0.0.0.0' then
+      return routes[i].nexthop
+    end
+  end
+  return nil
+end
+
+-- collect device data
+network_status = ubus:call('network.device', 'status', {})
+wireless_status = ubus:call('network.wireless', 'status', {})
+interface_data = ubus:call('network.interface', 'dump', {})
+nixio_data = nixio.getifaddrs()
+
+
```

Comment:
there's 2 blank lines but it seems to me we're using 1 blank line almost everywhere in this file